src/app/charts/content/cminfo.yaml

```diff
@@ -28,3 +28,14 @@ s3:
     region: us-east-1
   aliases:
     - minio
+
+box:
+  url: https://arkcase-demo.account.box.com
+  ui: https://arkcase-demo.account.box.com/login
+  users:
+    admin: box.admin@armedia.com
+  groups:
+    admin:
+      - box-administrators # is it the correct group?
```

Comment:
You need to add a `main` user as well that will be used by the application, even if it's the same `admin` user. The reason is simple: the `main` user is meant to be used by the application for normal operations, while the `admin` user is only to be used if/when administrative tasks are to be executed by the application.